components/notify_admin_cta/notify_admin_cta.tsx

```diff
@@ -38,13 +40,18 @@ export enum DafaultBtnText {
     Failed = 'Try again later!',
 }
 
-function NotifyAdminCTA() {
+type CTAProps = {
```

Comment:
Nit: Our convention is to name it as `Props` unless there is more than one component in a file. And if there is more than one component, we normally would want to split them out, one per file.